package/base-files/files/lib/functions/system.sh

```diff
@@ -66,7 +66,7 @@ get_mac_ascii() {
 	local key="$2"
 	local mac_dirty
 
-	mac_dirty=$(strings "$part" | sed -n 's/^'"$key"'=//p')
+	mac_dirty=$(strings "$part" | sed -n 's/^'"$key"'=//p' | head -1)
```

Comment:
hmmmm should be head -n 1 I think.

head -1 can be disabled for busybox IIRC.

Reply:
another issue:
on some case, the data `Ffac_mac = XX:XX:XX:XX:XX:XX`
then the get_mac_ascii() get nothing

Reply:
> another issue: on some case, the data `Ffac_mac = XX:XX:XX:XX:XX:XX` then the get_mac_ascii() get nothing

Fixed in https://github.com/openwrt/openwrt/pull/17262/commits/bc596b5f751c4ad66b28af24709ae7a03c7fe364.